plugins/verifier/licensechecker/licensechecker.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/deislabs/ratify/plugins/verifier/licensechecker/utils"
+
+	"github.com/deislabs/ratify/pkg/common"
+	"github.com/deislabs/ratify/pkg/ocispecs"
+	"github.com/deislabs/ratify/pkg/referrerstore"
+	_ "github.com/deislabs/ratify/pkg/referrerstore/oras"
+	"github.com/deislabs/ratify/pkg/verifier"
+	"github.com/deislabs/ratify/pkg/verifier/plugin/skel"
+)
+
+type PluginConfig struct {
+	Name            string   `json:"name"`
+	AllowedLicenses []string `json:"allowedLicenses"`
```

Comment:
are you planning to restrict the other way around too (disallowedlicenses)?

Reply:
I hadn't thought of that initially, but I think this would be a good add. I'll work on getting that in here as well.

Reply:
Added #125 to expand the functionality in the future. I would like to get this in and iterate on it.